egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -105,6 +111,49 @@ def GetSuccessfulModels(num_models, log_file_pattern, difference_threshold=1.0):
 
     return [accepted_models, max_index+1]
 
+def GetAverageNnetModel(dir, iter, nnets_list, run_opts, use_raw_nnet = False, shrink = None):
+    scale = 1.0
+    if shrink is not None:
```

Comment:
I haven't done any experiment on this. I used this since it is tested in acoustic model training.